lintcheck/src/config.rs

```diff
@@ -46,6 +46,11 @@ pub(crate) struct LintcheckConfig {
     /// Run clippy on the dependencies of crates specified in crates-toml
     #[clap(long, conflicts_with("max_jobs"))]
     pub recursive: bool,
+    /// Also produce a `perf.data` file, implies --jobs=1,
```

Comment:
Doesn't look like this `implies` is true currently, there might be a `#[clap]` flag for it though

Reply:
I realized it immediately after uploading the issue, just updated it to actually imply (declare a default value, and exit if it the flags don't comply)